apps/coscrad-frontend/src/utils/generic-components/presenters/detail-views/resource-detail-full-view-presenter.tsx

```diff
@@ -23,17 +26,37 @@ export interface ResourceDetailFullViewPresenterProps {
 export const ResourceDetailFullViewPresenter = ({
     id,
     name,
+    type,
     children,
 }: ResourceDetailFullViewPresenterProps): JSX.Element => (
     <div data-testid={id}>
         <Card>
-            <MultilingualTextPresenter text={name} />
-            {/* <CardMedia>
-            // We need to conditionally render media if they are specified
-            // Note that you are not supposed to have a CardMedia section with no children
-            {isNonEmptyString(audioUrl) ? <MediaPlayer audioUrl={audioUrl} /> : null}
-        </CardMedia> */}
-            <CardContent>{children}</CardContent>
+            <CardContent>
+                <Grid container spacing={1} columns={{ xs: 2, sm: 4, md: 12 }}>
+                    <Grid item xs={2} sm={1} md={2}>
+                        {/* Temporary.  We'd like an icon if there's no visual media associated with this resource */}
+                        {type !== ResourceType.photograph && (
```

Comment:
Note that this hack is forcing you to pass through props that we probably don't want as part of the API.